.env.example

```diff
@@ -7,6 +7,6 @@ SUBMIT_URL=https://submission.covidshield.app
 RETRIEVE_URL=https://retrieval.covidshield.app
 HMAC_KEY=aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
 MCC_CODE=302
-
 TEST_MODE=false
 MOCK_SERVER=false
+REGION_JSON_URL=
```

Comment:
Should this value have an example URL?

Reply:
Was debating this with myself.  Added now.